aws_inventory/invoker.py

```diff
@@ -49,6 +49,7 @@ def _probe_services(self):
             # create a config for all clients to share
             client_config = botocore.config.Config(
                 connect_timeout=config.CLIENT_CONNECT_TIMEOUT,
+                max_pool_connections=25,
```

Comment:
Why 25? Is there a way to derive a good value based off a client's runtime environment?

Reply:
It should be added to `config.py` first then set in the `invoker.py` as

``` python
max_pool_connections=config.CLIENT_MAX_POOL_CONNECTIONS
```